<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamingScoll</class>
 <widget class="QWidget" name="gamingScoll">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="pause" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>281</width>
     <height>251</height>
    </rect>
   </property>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="healthBar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="level">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>551</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QPushButton" name="continue_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>161</width>
      <height>93</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#continue_button{
background-color:rgba(0,0,0,0);
background-image:url(&quot;:/new/prefix2/icon/start_button.png&quot;);
}
QPushButton#continue_button:hover{
background-image:url(&quot;:/new/prefix2/icon/start_button_pressed.png&quot;);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="getReady">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>300</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(&quot;:/new/prefix2/icon/getReady.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="gameOver" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>551</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QLabel" name="game_over">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>300</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(&quot;:/new/prefix2/icon/GameOver.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="last_score">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>SCORE:</string>
    </property>
   </widget>
  </widget>
  <zorder>lcdNumber</zorder>
  <zorder>healthBar</zorder>
  <zorder>level</zorder>
  <zorder>widget</zorder>
  <zorder>gameOver</zorder>
  <zorder>pause</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
